rails_event_store_active_record/lib/rails_event_store_active_record/mappers/yaml.rb

```diff
@@ -0,0 +1,30 @@
+module RailsEventStoreActiveRecord
+  module Mappers
+    class YAML
+      def initialize
+        @serializer = ::YAML
+      end
+
+      def event_to_record(event)
+        Event.new(
```

Comment:
@andrzejsliwa nah, the mapper cannot assume the persistence layer. It might get stored in mongo. That's why I said we shold go with 4 separate attributes or 1 intermediary

Reply:
c'mon its implemented on specific repository, so can assume it for sure

Reply:
Ah.... I did not notice. It should be part of `ruby_event_store`, not `raills_event_store_active_record`.